.travis.yml

```diff
@@ -4,7 +4,11 @@ node_js:
   - 8
   - 10
 
+services:
```

Comment:
If we want to introduce e2e tests, we also need to provide a simple way to test locally, though.

Maybe something similar to how it's done in `pg-listen`?
https://github.com/andywer/pg-listen/blob/c34d0e1646d4b1f0f88cf021611e07cf05ad4ee5/package.json#L13

Reply:
Sure!